ui/imports/utils/Utils.qml

```diff
@@ -1004,4 +1004,8 @@ QtObject {
         return StatusQUtils.AmountsArithmetic.times(StatusQUtils.AmountsArithmetic.fromString(value),
                                        StatusQUtils.AmountsArithmetic.fromNumber(1, Constants.ethTokenGWeiDecimals))
     }
+
+    function generateAlias(pk) {
+        return globalUtilsInst.generateAlias(pk);
+    }
```

Comment:
Do we need that? In general it's totally against the guideline. Singletons should not call backend at all, moreover raw nim-injected context properties should be used only from stores.